src/components/AppContent/AppContent.js

```diff
@@ -1,17 +1,30 @@
-import React from "react";
+import React, { useState } from "react";
 import { Content } from "carbon-components-react";
 import Card from "../Card";
 
-const AppContent = () => (
-  <Content>
-    <div className="bx--grid">
-      <div className="bx--row">
-        <div className="bx--col-sm-6 bx--col-md-4 bx--col-lg-3">
-          <Card />
+const AppContent = (props) => {
+  const [contentState] = useState(props.appState);
+
+  function contentStateHandler(data) {
+    props.appStateHandler((prevState) => {
+      return { ...prevState, ...data };
+    });
+  }
```

Comment:
What is the purpose of this method? Why just not pass appStateHandler to the card?